bool ldr1
bool ldr2
